db/migrations/migrations_test.go

```diff
@@ -86,6 +86,16 @@ var migration5Tables = []interface{}{
 	"suppression_files",
 }
 
+var migration7Tables = []string{
```

Comment:
Since we're only using this in the TestBCDAMigration() function, should we define this in that function?

Reply:
We can, it would make sense to put it there as well!